doc/integrator/filter.rst

```diff
@@ -3,8 +3,19 @@
 Filter (Querier)
 ================
 
+Available attributes and operators in filters
+---------------------------------------------
+
+All attributes defined as "exported" in the layer of you map server will be automatically
+available as filterable attribute. If the type, and so the operator on the attribute, is not
+adapted, you should adapt it manually in your layer definition.
```

Comment:
suggestion:
"If the type, and so the operator on the attribute, is not adequate for filtering, you should adapt the type in your layer definition."